src/features/ajax/instrument/index.js

```diff
@@ -93,7 +93,7 @@ function subscribeToEvents (agentIdentifier, ee, handler, dt) {
     ctx.xhrGuids = {}
     ctx.lastSize = null
     ctx.loadCaptureCalled = false
-    ctx.params = this.params || {}
+    ctx.params = this.params
```

Comment:
Is `this.params` never undefined? Or does it not matter if `ctx.params` isnt' an object?

Reply:
I think this was an accidental removal.  Will add back in